substrate/utils/frame/remote-externalities/src/lib.rs

```diff
@@ -58,37 +58,37 @@ type ChildKeyValues = Vec<(ChildInfo, Vec<KeyValue>)>;
 type SnapshotVersion = Compact<u16>;
 
 const LOG_TARGET: &str = "remote-ext";
-const DEFAULT_HTTP_ENDPOINT: &str = "https://rpc.polkadot.io:443";
-const SNAPSHOT_VERSION: SnapshotVersion = Compact(3);
+const DEFAULT_HTTP_ENDPOINT: &str = "https://polkadot-try-runtime-node.parity-chains.parity.io:443";
```

Comment:
`https://rpc.polkadot.io:443` is not usable anymore due to increased rate-limiting.

Reply:
Not sure it helps much to have a default when this can be used on any chain?

Reply:
It's nice to not require specifying it when you just want to run the tests and don't care about which chain